### ISSUE TITLE
NTR: Spleen biopsy: lipid-laden macrophages (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9214]
New term request Spleen biopsy: lipid-laden macrophages (elthat_101212171048): Splenic foamy cells  UMLS: 1859005

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000160
- label: Splenic lipid-laden macrophage infiltration
- parents: [Abnormal spleen morphology (HP:0025408)](https://hpo.jax.org/app/browse/term/HP:0025408)
- pmid: PMID:32382506
- def: Accumulation of foamy cells (lipid-laden macrophages) in the parenchyma of the spleen.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9214
- synonyms: Spleen biopsy: lipid-laden macrophages(NONE)|Splenic foamy cells(NONE)
